experimental/overhead_matching/swag/model/panorama_semantic_landmark_extractor.py

```diff
@@ -130,21 +177,33 @@ def load_files(self):
                 self.panorama_metadata.update(new_metadata)
                 assert len(self.panorama_metadata) == old_metadata_size + new_pano_metadata_len
 
+        # Load the semantic class groupings
```

Comment:
duplicate of above?

Reply:
Should have been removed. Will do before merging